src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/src/api-authorization/login-menu/login-menu.component.html

```diff
@@ -6,7 +6,7 @@
         <a  class="nav-link text-dark" [routerLink]='["/authentication/logout"]' [state]='{ local: true }' title="Logout">Logout</a>
     </li>
 </ul>
-<ul class="navbar-nav" *ngIf="!(isAuthenticated | async)">
+<ul class="navbar-nav" *ngIf="(isAuthenticated | async) === null">
```

Comment:
@javiercn this was the only change I had to make to get the individual auth changes to be happy.  There was a lint warning about using bool, not sure what this is doing, but it suggested to explicitly === against null | false | undefined.  False didn't work, but null seems to preserve the old behavior.  But you should double check this magic :)

Reply:
@HaoK `!== true` might be a better choice, since that would work for `null | undefined | false`